install/kubernetes/cilium/values.yaml

```
@@ -756,6 +756,30 @@ hubble:
     # -- Whether to enable the Hubble UI.
     enabled: false
 
+    standalone:
+      # -- When true, it will allow installing the Hubble UI only, without checking dependencies.
+      # It is useful if a cluster already has cilium and Hubble relay installed and you just
+      # want Hubble UI to be deployed.
+      # When installed via helm, installing UI should be done via `helm upgrade` and when installed via the cilium cli, then `cilium hubble enable --ui`
+      enabled: false
+
+      tls:
+        # -- When deploying Hubble UI in standalone, with tls enabled for Hubble relay, it is required
+        # to provide a volume for mounting the client certificates.
+        certsVolume: null
+          # projected:
+          #   defaultMode: 420
```

use the same mode as the Helm default

```suggestion
          #   defaultMode: 0400
```